File: src/main/java/appjoe/wordpress/com/testdemo/ApiService.java
package appjoe.wordpress.com.testdemo;


import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiService {
    String BASE_URL = "http://www.androidbegin.com/";

//    @GET("tutorial/jsonparsetutorial.txt")
//    void getPopulationData(Callback<JsonResponse> callback) ;

    @GET("tutorial/jsonparsetutorial.txt")
    Call<JsonResponse> getPopulationData();
}

File: src/main/java/appjoe/wordpress/com/testdemo/ApiService.kt
package appjoe.wordpress.com.testdemo


import retrofit2.Call
import retrofit2.http.GET

interface ApiService {

    //    @GET("tutorial/jsonparsetutorial.txt")
    //    void getPopulationData(Callback<JsonResponse> callback) ;

    @get:GET("tutorial/jsonparsetutorial.txt")
    val populationData: Call<JsonResponse>

    companion object {
        val BASE_URL = "http://www.androidbegin.com/"
    }
}
